speedtest.go

```diff
@@ -37,38 +38,47 @@ func main() {
 		return
 	}
 
-	if *showCityList {
-		speedtest.PrintCityList()
-		return
-	}
+	var targets speedtest.Servers
+	if *customURL == "" {
```

Comment:
Could you move this `if` statement to L66?
With the current condition, `showUser(user)` method won't be called when we use `--custom-url` option.
and this is minor reason but I'd like to minimize the scope of if-else condition 👍

Reply:
Sounds great, adjusting that now, ty!